app/views/articles/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Edit article</h1>
+
+<%= form_with model: @article do |form| %>
+  <%= form.label :title %>
+  <%= form.text_field :title, value: @article.title, placeholder: 'Title' %>
```

Comment:
Do you really need a `label` AND a `placeholder` on this field (and the one below)? Feels a bit redundant? Like 'hänglen och livrem' if you know what I mean... ;-)